examples/multiple-choice/index.html

```diff
@@ -41,8 +41,8 @@
       [
         {
           question: 'What framework should we start with?',
-          answers: ['React', 'Ember.js', 'Vue'],
-          answer: 1
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'jQuery'],
```

Comment:
@pachonk have some visual existing example? As @tmornini says... I see the words... just having some issues putting the sentences together. /cc @albertoponti